core/src/config.rs

```diff
@@ -114,12 +114,12 @@ struct Cli {
     /// The HTTP addressable StarkNet JSON-RPC node
     #[clap(long = "starknet-gateway", value_parser)]
     pub starknet_jsonrpc_node: Option<String>,
-    /// The StarkNet address corresponding to the given key
-    #[clap(long = "starknet-account-addr", value_parser, requires = "starknet-private-key")]
-    pub starknet_account_address: Option<String>,
-    /// The StarkNet private key used to send transactions
-    #[clap(long = "starknet-account-pkey", value_parser, requires = "starknet-account-address")]
-    pub starknet_private_key: Option<String>,
+    /// The StarkNet addresses corresponding to the given private keys
+    #[clap(long = "starknet-account-addr", value_parser, requires = "starknet-private-keys")]
+    pub starknet_account_addresses: Option<Vec<String>>,
+    /// The StarkNet private keys used to send transactions
+    #[clap(long = "starknet-account-pkey", value_parser, requires = "starknet-account-addresses")]
+    pub starknet_private_keys: Option<Vec<String>>,
```

Comment:
Can we just make these `Vec<String>`? No need for the option anymore, it can just be empty

Reply:
same everywhere else they're used